examples/models/llama2/runner/runner.h

```diff
@@ -41,7 +41,11 @@ class Runner {
       std::function<void(const std::string&)> token_callback = {},
       std::function<void(const ::executorch::extension::llm::Stats&)>
           stats_callback = {},
-      bool echo = true);
+      bool echo = true,
+      bool warming = false);
+  ::executorch::runtime::Error warmup(
+      const std::string& prompt,
+      int32_t seq_len = 128);
```

Comment:
should we move seq_len out to a constant variable DEFAULT_SEQ_LEN that is shared by both generate and warmup?

Reply:
so warmup is done on the same prompt/seq_len as a real run today. Default is just something I followed from generate().